frontend/apps/reader/modules/readerview.lua

```diff
@@ -504,10 +561,16 @@ function ReaderView:drawPageSavedHighlight(bb, x, y)
                 local pos0, pos1 = item.pos0, item.pos1
                 local boxes = self.document:getPageBoxesFromPositions(page, pos0, pos1)
                 if boxes then
+                    local drawer = item.drawer or self.highlight.saved_drawer
+                    local is_note_mark = self.highlight.note_mark and
```

Comment:
This variable name `is_note_mark`, here and below, sounds it would be better named `draw_note_mark`